{"error": {"@Message.ExtendedInfo": [{"@odata.type": "#Message.v1_0_5.Message", "Message": "The request failed due to an internal service error.  The service is still operational.", "MessageArgs": [], "MessageId": "Base.1.1.InternalError", "RelatedProperties": [], "Resolution": "Resubmit the request.  If the problem persists, consider resetting the service.", "Severity": "Critical"}], "code": "Base.1.1.InternalError", "message": "The request failed due to an internal service error.  The service is still operational."}}